elastic/logs/track.json

```diff
@@ -42,9 +42,10 @@
 {% set p_restore_data_streams = (restore_data_streams | default("logs-*")) %}
 {% set p_snapshot_metadata = (snapshot_metadata | default({}))%}
 {% set p_snapshot_rename_suffix = (snapshot_rename_suffix | default("") ) %}
+{% set p_snapshot_restore_counts = (snapshot_restore_counts | default(100) ) %}
 {% set p_dsl_poll_interval = (dsl_poll_interval | default(false) ) %}
 {% set p_dsl_default_rollover = (dsl_default_rollover | default(false) ) %}
-
+{% set p_skip_fleet_globals = (skip_fleet_globals | default(false) ) %}
```

Comment:
I just copied this from another PR. Will eventually remove after the other PR is merged.
This is necessary to avoid an error which happens when deleting component templates.